### PR TITLE
Added Astro Config and Vercel Icons

### PR DESCRIPTION
**Pull request by max-programming**
Let me know if any changes are required.

> P.S. I haven't tested it out because idk how I can run this extension but I guess it will work

### CODE REVIEW

File: src/data/languages.json [hunk 1]
```diff
@@ -344,6 +350,9 @@
         "/jest.config\\.(js|ts)/i": {
             "image": "jest"
         },
+        "/tailwind.config\\.(js|cjs)/i": {
```
Comment (xhayper) [inline src/data/languages.json:353]: `/tailwind\\.config\\.(js|cjs)/i`

File: src/data/languages.json [hunk 2]
```diff
@@ -323,12 +323,18 @@
         "turbo.json": {
             "image": "turbo"
         },
+        "/(vercel|now).json/i": {
```
Comment (xhayper) [inline src/data/languages.json:326]: `/(vercel|now)\\.json/i`

**Review by xhayper — CHANGES_REQUESTED** [review]
Just a few regex change! and you will be good to go!

**Comment by max-programming** [comment]
Pushed the changes but for some reason it didn't reflect here

**Comment by max-programming** [comment]
Done!

**Review by xhayper — APPROVED** [review]

**Pull request merged**